apps/ui/src/Pages/RidesFeed/CreateRideDialog.tsx

```diff
@@ -150,7 +168,9 @@ const CreateRideDialog = ({
           type="number"
           fullWidth
           value={gasMoney}
-          onChange={(e) => setGasMoney(e.target.value)}
+          onChange={(e) => handleGasSeatsChange('gasMoney', e.target.value)}
+          error={isGasMoneyNegative}
+          helperText={isGasMoneyNegative ? 'Gas Money cannot be negative' : ''}
```

Comment:
use consts instead with useState or useMemo

Reply:
simple sentence decided to leave it here